.eslintrc.json

```diff
@@ -7,6 +7,7 @@
 	"parser": "@babel/eslint-parser",
 	"rules": {
 		"indent": "off",
-		"operator-linebreak": "off"
+		"operator-linebreak": "off",
+		"linebreak-style": 0
```

Comment:
We can still merge with this but we'll all have to disable it I'm afraid. 
Raj pointed out it's not a great idea. That's my bad.